go/vt/servenv/status.go

```diff
@@ -337,4 +338,8 @@ func init() {
 	if err != nil {
 		log.Exitf("os.Hostname: %v", err)
 	}
+
+	_ = stats.NewGaugeFunc("Uptime", "Uptime in seconds", func() int64 {
+		return int64(time.Since(serverStart).Seconds())
```

Comment:
why not something finer-grained like millis, or even nanos? consumers can always truncate/divide and lose precision, but they can never get it back if we clip it here

Reply:
Update: there is actually a mix of nanosecond-based variables with and without `Ns` as a prefix, examples: `TransactionTimeout` and `QueryTimeout` (both nanosecond-based). It seems most of the metrics with `Ns` suffix are related to histograms and operation timings

So if no objections I'll stick with the name `Uptime` as it seems a bit more friendly 👍